generator/intersection.go

```diff
@@ -0,0 +1,76 @@
+package generator
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
```

Comment:
case-insensitive import collision

```suggestion
	"github.com/Sirupsen/logrus"
```

Reply:
Done